### DISCUSSION TITLE
Problem Loading Custom Themes

### DISCUSSION THREAD

**Discussion by avalon60** [TomSchimansky/CustomTkinter#442] [Q&A] [answered]
Hello.

Is there a problem with loading custom themes from outside of the site-packages location?

Not sure if I am doing something wrong, but I get this:
`
FileNotFoundError: [Errno 2] No such file or directory: 'Oceanix.json'`

Looking at this through the debugger, I see this:

![image](https://user-images.githubusercontent.com/89534395/189493460-bfe1af9b-40c1-401e-9164-a2e557aff1f0.png)

I verify that this is the location of my JSON:

![image](https://user-images.githubusercontent.com/89534395/189493510-e7bd3414-22b1-4dc2-8371-61706955bb41.png)

Also I see that by the time we get to the Thememanager code we see this:

![image](https://user-images.githubusercontent.com/89534395/189493586-5a281e1f-1293-46b0-9d5c-c66bb88e3f53.png)

So it looks as though it's trying to open a file, without the full pathname and so fails.

Thanks,

Clive

**Comment by Akascape** [comment, marked as answer]
@avalon60 I am not getting any error while loading external themes, 

**I directly use this method for loading it:**
`customtkinter.set_default_color_theme("themepath/my_theme.json")`

You have to specify the full path including the .json extension.

If you want to add the theme in **site packages,** then first paste the my_theme.json file inside the `customtkinter>assets>themes` folder and then add the new theme name in this [list](https://github.com/TomSchimansky/CustomTkinter/blob/f39ee5764a2979f0c56d15a7da2c1bd4f6cdd838/customtkinter/theme_manager.py#L9) of theme_manager. 

**Like this:** 
`built_in_themes = ["blue", "green", "dark-blue", "sweetkind", "my_theme"]`
_No need to add the .json file extension._

Then you can use it directly like this:
`customtkinter.set_default_color_theme("my_theme")`

I don't know if it will help or not...

**Reply by avalon60** [reply]
Thanks @Akascape - I'll take another look, looking at the screenshot now, I think I spotted the bug which is in my code!
